lib.php

```diff
@@ -7316,7 +7316,7 @@ function hsuforum_cm_info_view(cm_info $cm) {
         $out .= '</a>';
     }
 
-    $cm->set_content($cm->content . $out); // append the unreadpost section to existing content
+    $cm->set_content($cm->get_formatted_content() . $out); // append the unreadpost section to existing content
```

Comment:
Are we 100% sure that the function "get_formatted_content" returns the same content as what should be in the content object attribute?

Reply:
The content attribute is private and in no way am I able to be 100% sure about this without deploying a test and getting someone to test, or testing it myself. However, this line is the only reason for all of the changes in this file as the $content attribute throws an issue in datadog due to it being undefined.